### ISSUE TITLE
Failed to install sumo-tools on Ubuntu 20.04

### ISSUE THREAD

**Issue by xxArbiter** [eclipse/sumo#8254]
Dear SUMO developers,

We are trying to install SUMO on a new server. While executing the command `sudo apt-get install sumo sumo-tools sumo-doc`, an error arose:

Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 sumo-tools : Depends: python
E: Unable to correct problems, you have held broken packages.

We tried python2.7, 3.6, and 3.8 but it still won't work. Could you help us figure this issue out? Thanks!

**Comment by ncarrara** [comment]
to be specific, the problem only comes from sumo-tools package

**Comment by behrisch** [comment]
ubuntu 20.04 does not provide a python package. We need to change the dependency and probably update all shebangs. There is currently no workaround

**Comment by xxArbiter** [comment]
I compiled the SUMO from the source code. And that works for our case.

**Comment by behrisch** [comment]
The problem is that ubuntu 20.04 does not provide the python package any longer (only python2 or python3). I tried to update the package removing all references to python. Please retry.

**Comment by behrisch** [comment]
Closing for lack of feedback. Please reopen if problems persist.

**Comment by xxArbiter** [comment]
Hi, sorry for such a late update. We compiled the source code at that time which works fine.

Recently we revisit this issue, tried the apt-get install. And the error is still there.

I tried these commands:
`sudo add-apt-repository ppa:sumo/stable`
`sudo apt-get update`
`sudo apt-get install sumo sumo-tools sumo-doc`

And still, get the unmet dependencies message. Is there something we missed? Thanks!
